Packs/Mimecast/Integrations/MimecastV2/README.md

```diff
@@ -9,8 +9,8 @@ Please read detailed instructions in order to understand how to set the integrat
 
```

Comment:
you should add -

1. The new command
2. Update the deprecate commands
3. Instruction about the api 2.0
4. Rate limit section.